policy/modules/system/libraries.if

```diff
@@ -531,3 +531,21 @@ interface(`libs_relabel_shared_libs',`
 
 	relabel_files_pattern($1, lib_t, { lib_t textrel_shlib_t })
 ')
+
+########################################
+## <summary>
+##	watch lib dirs
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+#
+interface(`libs_watch_shared_libs_dir',`
```

Comment:
Please make the "dir" plural, i.e. `libs_watch_shared_libs_dirs`

Reply:
done